[2025-02-02T20:10:03.908+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T20:10:03.951+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_processing.schema_create scheduled__2025-02-02T20:08:00+00:00 [queued]>
[2025-02-02T20:10:03.968+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_processing.schema_create scheduled__2025-02-02T20:08:00+00:00 [queued]>
[2025-02-02T20:10:03.968+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-02T20:10:04.003+0000] {taskinstance.py:2889} INFO - Executing <Task(SQLExecuteQueryOperator): schema_create> on 2025-02-02 20:08:00+00:00
[2025-02-02T20:10:04.017+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1600) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T20:10:04.020+0000] {standard_task_runner.py:72} INFO - Started process 1608 to run task
[2025-02-02T20:10:04.021+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_processing', 'schema_create', 'scheduled__2025-02-02T20:08:00+00:00', '--job-id', '714', '--raw', '--subdir', 'DAGS_FOLDER/DAG_etl.py', '--cfg-path', '/tmp/tmpko46my_4']
[2025-02-02T20:10:04.026+0000] {standard_task_runner.py:105} INFO - Job 714: Subtask schema_create
[2025-02-02T20:10:04.134+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_processing.schema_create scheduled__2025-02-02T20:08:00+00:00 [running]> on host a664bcf281bf
[2025-02-02T20:10:04.697+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_processing' AIRFLOW_CTX_TASK_ID='schema_create' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T20:08:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-02T20:08:00+00:00'
[2025-02-02T20:10:04.699+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T20:10:04.737+0000] {sql.py:278} INFO - Executing: 
    create schema if not exists nds;
    create schema if not exists dds;
    create schema if not exists stg;
[2025-02-02T20:10:04.753+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:10:04.770+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:10:04.776+0000] {sql.py:544} INFO - Running statement: 
    create schema if not exists nds;
    create schema if not exists dds;
    create schema if not exists stg;, parameters: None
[2025-02-02T20:10:04.806+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T20:10:04.806+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_processing, task_id=schema_create, run_id=scheduled__2025-02-02T20:08:00+00:00, execution_date=20250202T200800, start_date=20250202T201003, end_date=20250202T201004
[2025-02-02T20:10:04.889+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T20:10:04.917+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
